src/rules/javascript/low_level_module.js

```diff
@@ -0,0 +1,48 @@
+import { LOW_LEVEL_MODULES } from 'const';
+import { LOW_LEVEL_MODULE } from 'messages';
+import { getVariable } from 'utils';
+
+
+/*
+ * This rule will detect use of `require()` with the first arg being either
+ * a literal that matches a restricted module or a var pointing to a literal
+ * that matches a restricted value.
+ *
+ * TODO: This rule should only be run for jetpack.
+ * TODO: Check what the requires_chrome feature does in the old code.
+ *
+ */
+export default function(context) {
+  return {
+    CallExpression: function(node) {
+      var requiresLowLevelMod = false;
+      if (node.callee.name === 'require' &&
+          node.arguments &&
+          node.arguments.length) {
+
+        var firstArg = node.arguments[0];
+
+        // Find a literal string value passed to the
+        // the require function.
+        if (firstArg.type === 'Literal' &&
+            LOW_LEVEL_MODULES.indexOf(firstArg.value) > -1) {
```

Comment:
Oops, yeah, I think contains was a Moz-only thing. If we have it we should use it (just clearer) but otherwise the PR is r+ from me!